scalableminds/webknossos
# Gracefully handle unsupported voxel types

- [X] Backend part is done
- [X] Dataset viewing needs appropriate error message for unsupported data types
- [x] WKW lib needs to be published

### Steps to test:
- you need the changes in wkw, https://github.com/scalableminds/webknossos-wrap/pull/30 (for now, checkout that and perform `sbt publishLocal` in the `scala` folder)
- unpack the [attached test dataset](https://github.com/scalableminds/webknossos/files/2796482/signed-int-test-dataset.tar.gz) to your binaryData directory
- you should be able to import it, it should be listed as having an int8 color layer
- (viewing does not yet work, @philippotto do you want to have a look at displaying an appropriate error message?)

### Issues:
- fixes #3608 

------
